backstop/tests/button.html

```diff
@@ -110,7 +110,7 @@ <h3>High visibility</h3>
 
         <button class="iui-button iui-high-visibility iui-small">
           <svg-placeholder
-            class="iui-icon"
+            class="iui-button-icon "
```

Comment:
There's extra spaces everywhere. Recommend manually formatting with prettier and then saving (which will format again with vscode formatter).